api/core/workflow/nodes/question_classifier/question_classifier_node.py

```diff
@@ -46,6 +47,13 @@ def _run(self):
         # extract variables
         variable = variable_pool.get(node_data.query_variable_selector) if node_data.query_variable_selector else None
         query = variable.value if variable else None
+        if query:
```

Comment:
We've no plan to support files in the Question Classifier node. If you just want to convert the input to string, you can use `variable.text` directly.